.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default code owners
+* @openebs/openebs-reviewers
+
+# CODEOWNERS file owner
+/.github/CODEOWNERS @openebs/org-maintainers
```

Comment:
```

CODEOWNERS errors
Unknown owner on line 5: make sure the team @openebs/org-maintainers exists, is publicly visible, and has write access to the repository
/.github/CODEOWNERS @openebs/org-maintainers

```

I see this flagged on the files. What's wrong here?

Reply:
I see `/+github/CODEOWNERS` instead of `/.github/CODEOWNERS` when rendered.

Reply:
Looks like the diff + is merged into the line. GitHub render bug most likely.

Reply:
I see this error:
```
Unknown owner on line 5: make sure the team @openebs/org-maintainers exists, is publicly visible, and has write access to the repository
```